### ISSUE TITLE
Voiceover doesn't read toast message on iOS

### ISSUE THREAD

**Issue by DK15** [flutter/flutter#41080]
If an app has Flutter toast message, then the talkback on Android reads it as expected, but voiceover on iOS doesn't read the message.

On button tap, I am showing a simple toast message as:
```
void _showToastMessage() {
    Fluttertoast.showToast(
        msg: "This is Long Toast",
        toastLength: Toast.LENGTH_LONG,
        gravity: ToastGravity.BOTTOM,
        timeInSecForIos: 1,
        backgroundColor: Colors.red,
        textColor: Colors.white,
        fontSize: 16.0
    );
  }
```
When talkback is turned ON, on Android emulator (Nexus 5X, 8.0), it reads the message properly as `{This is Long Toast}`

But on iOS (iPhone 6s, 11.2.1), voiceover doesn't read this message.

## Flutter doctor:

```
[✓] Flutter (Channel stable, v1.7.8+hotfix.3, on Mac OS X 10.13.6 17G65, locale en-US)
    • Flutter version 1.7.8+hotfix.3 at /Users/deeptibelsare/Documents/Flutter-sdk/flutter
    • Framework revision b712a172f9 (2 months ago), 2019-07-09 13:14:38 -0700
    • Engine revision 54ad777fd2
    • Dart version 2.4.0

[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/deeptibelsare/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit https://flutter.dev/setup/#android-setup for detailed instructions.

[✓] Xcode - develop for iOS and macOS (Xcode 9.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.4.1, Build version 9F2000
    • CocoaPods version 1.7.0.beta.3

[!] iOS tools - develop for iOS devices
    • ios-deploy 2.0.0
    ✗ ios-deploy out of date (1.9.4 is required). To upgrade with Brew:
        brew upgrade ios-deploy

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 34.0.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.0.0 (API 26) (emulator)

! Doctor found issues in 2 categories.
```

**Comment by janmoppel** [comment]
Hello @DK15 !

Thank you for posting an issue.
Can you please create a self-contained repository, so that your issue could be easily reproduced? It would be very helpful.

Thank you.

**Comment by DK15** [comment]
@janmoppel I updated issue with required details. Hope it helps.

**Comment by DK15** [comment]
I observed that unlike Android, on iOS we need to explicitly select the toast message in order for voiceover to read it. Hence, I am closing this ticket as non-issue.
